src/gui/windows/chat.zig

```diff
@@ -29,6 +31,7 @@ pub var window: GuiWindow = GuiWindow{
 const padding: f32 = 8;
 const messageTimeout: i32 = 10000;
 const messageFade = 1000;
+const reusableHistoryMaxSize = 128;
```

Comment:
This memory costs nothing, but if a user does hit the "end of history", then it is rather painful (trust me I've been there)
I'd at least set it to 10000 or something.

Reply:
Increased to 8192, I like multiples of 2